Attempted to deploy Azure Policy Assignment but the plugin crashed.

**Azure Provider Version**
```
Initializing provider plugins...
- Finding latest version of hashicorp/azurerm...
- Installing hashicorp/azurerm v2.94.0...
- Installed hashicorp/azurerm v2.94.0 (signed by HashiCorp)
```
**Terraform Version**
v1.0.0

**Trace of Plugin problem**
```
Stack trace from the terraform-provider-azurerm_v2.94.0_x5 plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 190 [running]:
github.com/hashicorp/terraform-provider-azurerm/internal/services/policy.expandAzureRmPolicyNotScopes({0xc00154f930, 0x1, 0x11b})
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/services/policy/policy_assignment_resource.go:420 +0x189
github.com/hashicorp/terraform-provider-azurerm/internal/services/policy.assignmentBaseResource.createFunc.func1({0x71be1a8, 0xc00202af60}, {0xc000bc2700, {0x71bf8d8, 0xc00000d440}, 0xc000067780, 0x0, {0x71c15b8, 0xb7e9ea0}})
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/services/policy/assignment_base_resource.go:88 +0xa92
github.com/hashicorp/terraform-provider-azurerm/internal/sdk.(*ResourceWrapper).Resource.func2({0x71be1a8, 0xc00202af60}, 0x1a3185c4b0e, {0x61cc880, 0xc000bc2700})
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/sdk/wrapper_resource.go:52 +0x176
github.com/hashicorp/terraform-provider-azurerm/internal/sdk.diagnosticsWrapper.func1({0x71be1a8, 0xc00202af60}, 0xc001c076c0, {0x61cc880, 0xc000bc2700})
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/sdk/wrapper_resource.go:185 +0xbe
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000439420, {0x71be138, 0xc001933300}, 0x2, {0x61cc880, 0xc000bc2700})
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:341 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000439420, {0x71be138, 0xc001933300}, 0xc000f0fc70, 0xc000067600, {0x61cc880, 0xc000bc2700})
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:467 +0x871
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc001368570, {0x71be138, 0xc001933300}, 0xc00145cdc0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/grpc_provider.go:977 +0xd8a
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc000067580, {0x71be1e0, 0xc000b9c2d0}, 0xc0001d5500)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server/server.go:603 +0x30e
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x6711e80, 0xc000067580}, {0x71be1e0, 0xc000b9c2d0}, 0xc00202a8a0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000d0fc0, {0x720ce08, 0xc0005fc000}, 0xc0015ebc20, 0xc0013d3230, 0xb7a9c80, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1292 +0xc6f
google.golang.org/grpc.(*Server).handleStream(0xc0000d0fc0, {0x720ce08, 0xc0005fc000}, 0xc0015ebc20, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1617 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:940 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:938 +0x294

Error: The terraform-provider-azurerm_v2.94.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```
<img width="1261" alt="Screen Shot 2022-02-01 at 12 22 35 PM" src="https://user-images.githubusercontent.com/54757326/152042419-afdd6784-7ef3-4d6a-8a39-2d61da2f4343.png">

--- a/internal/services/policy/policy_assignment_resource.go
+++ b/internal/services/policy/policy_assignment_resource.go
@@ -417,7 +417,10 @@ func expandAzureRmPolicyNotScopes(input []interface{}) *[]string {
 	notScopesRes := make([]string, 0)
 
 	for _, notScope := range input {
-		notScopesRes = append(notScopesRes, notScope.(string))
+		s, ok := notScope.(string)
+		if ok {
+			notScopesRes = append(notScopesRes, s)
+		}
 	}
 
 	return &notScopesRes

